pkg/util/kvgraph/backup_graph_test.go

```diff
@@ -330,7 +342,8 @@ func TestNewVirtualMachineBackupGraph(t *testing.T) {
 				},
 			},
 		},
-		{"Should include backend PVC",
+		{
+			"Should include backend PVC",
```

Comment:
why this changes?

Reply:
apologies nvim-conform strikes again, I'll repsin and remove this.